src/modal/modal.js

```diff
@@ -76,24 +70,16 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
 
     function linkFn(scope, element, attrs) {
       if (attrs.modalInClass) {
-        if ($animateCss) {
-          $animateCss(element, {
-            addClass: attrs.modalInClass
-          }).start();
-        } else {
-          $animate.addClass(element, attrs.modalInClass);
-        }
+        $animateCss(element, {
```

Comment:
function animateIn(cssKey) {
  var animateObj = {};
  animateObj[cssKey] = attes.modalInClass;
  return $animateCss(element, animateObj).start();
}

Or something like that